message Chunk {

    required string cid = 1;
    required string sid = 2;

    required bytes content = 3;
    required uint32 initial_seq = 4;
    required uint32 ttl = 5;

    required int64 created_time = 6;
    required int64 accessed_time = 7;

}
